src/components/Select/Select.module.scss

```diff
@@ -0,0 +1,255 @@
+@import '../../styles/mixins';
```

Comment:
Please add blank lines to this file. After every block on the same level should be blank line.